packages/p/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@react-email/p",
+  "version": "0.0.1",
+  "description": "A paragraph is a block of text separated by some white space",
```

Comment:
Maybe we can omit the "A paragraph is.."

```suggestion
  "description": "A block of text separated by some white space",
```

Reply:
good point

Reply:
done!